src/engine/renderer/tr_init.cpp

```diff
@@ -830,8 +831,8 @@ ScreenshotCmd screenshotPNGRegistration("screenshotPNG", ssFormat_t::SSF_PNG, "p
 
 		tr.currenttextures.resize( glConfig2.maxTextureUnits );
 
-		GL_TextureMode( r_textureMode->string );
-		r_textureMode->modified = false;
+		GL_TextureMode( r_textureMode.Get().c_str() );
```

Comment:
This looks like it can be just `GL_TextureMode( r_textureMode.GetModifiedValue().c_str() );` since it should always be `modified` at this point?

Reply:
It will not always be modified. I believe this code runs on every map change.

Reply:
I see.